.gitignore

```diff
@@ -30,3 +30,4 @@
 .mindtagger.conf.json
 /out
 /log
+/lib
```

Comment:
I was thinking we don't want to track the library files. `library/` is also not included in the source tree, so we are currently not tracking it.